### PR TITLE
Update dependencies to latest versions. Fix _.get based on new @types/lodash

### PR DESCRIPTION
**Pull request by a-tarasyuk**

### CODE REVIEW

File: src/ComponentBase.ts [hunk 1]
```diff
@@ -391,6 +399,40 @@ export abstract class ComponentBase<P extends React.Props<any>, S extends Object
...
+     * search Subscription "keyPropertyName" in Component props(this.props)
+     *
+     * @private
+     * @param {Readonly<P>} props
+     * @param {string} keyPropertyName
+     *
+     * @return {string}
+     */
+    private _findKeyPropertyName(props: Readonly<P>, keyPropertyName: string): string {
+        const key: string | undefined = _.get(this.props, keyPropertyName);
```
Comment (berickson1) [inline src/ComponentBase.ts]: I think you meant to use props instead of this.props

File: src/ComponentBase.ts [hunk 2]
```diff
@@ -391,6 +399,40 @@ export abstract class ComponentBase<P extends React.Props<any>, S extends Object
...
+    /**
+     * search Subscription "keyPropertyName" in Component props(this.props)
+     *
+     * @private
+     * @param {Readonly<P>} props
+     * @param {string} keyPropertyName
+     *
+     * @return {string}
+     */
+    private _findKeyPropertyName(props: Readonly<P>, keyPropertyName: string): string {
```
Comment (berickson1) [inline src/ComponentBase.ts]: I'm making another change (just went up in PR) that should allow doing ```keyof P``` here and below

File: src/ComponentBase.ts (hunk 1 above)
Comment (a-tarasyuk) [reply src/ComponentBase.ts]: Yes, it is a typo. I fixed it.

File: src/ComponentBase.ts (hunk 2 above)
Comment (a-tarasyuk) [reply src/ComponentBase.ts]: Should I decline this PR?

**Pull request merged**